File: examples/src/Presentations/Conversion/ExportToHTMLWithResponsiveLayout.cs
import aspose.slides as slides
using Aspose.slides.Examples.CSharp
using Aspose.slides.Export
using System
using System.Collections.Generic
using System.Linq
using System.text
using System.Threading.Tasks

namespace CSharp.Presentations.Conversion
{
    class ExportToHTMLWithResponsiveLayout
    {
        public static void Run() {

            #ExStart:ExportToHTMLWithResponsiveLayout
            # The path to the documents directory.
            dataDir = RunExamples.GetDataDir_Conversion()

            Presentation presentation = new Presentation(dataDir+"SomePresentation.pptx")
            HtmlOptions saveOptions = new HtmlOptions()
            saveOptions.SvgResponsiveLayout = True
            presentation.save(dataDir+"SomePresentation-out.html", SaveFormat.Html, saveOptions)
            #ExEnd:ExportToHTMLWithResponsiveLayout
        }
    }
}

File: examples/src/Presentations/Conversion/ExportToHTMLWithResponsiveLayout.py
import aspose.slides as slides

def convert_to_html_with_responsive_layout():
    #ExStart:ExportToHTMLWithResponsiveLayout
    # The path to the documents directory.
    dataDir = "./examples/data/"
    outDir = "./examples/out/"

    with slides.Presentation(dataDir + "welcome-to-powerpoint.pptx") as presentation:
        saveOptions = slides.export.HtmlOptions()
        saveOptions.svg_responsive_layout = True
        presentation.save(outDir+"convert_to_html_with_responsive_layout_out.html", slides.export.SaveFormat.HTML, saveOptions)
    #ExEnd:ExportToHTMLWithResponsiveLayout
